cmds/webboot/utils.go

```diff
@@ -114,3 +114,19 @@ func validURL(url string) (string, string, bool) {
 		return url, "Invalid URL.", false
 	}
 }
+
+func fileExists(filepath string) bool {
```

Comment:
I'm wrong again .... I'm sorry.

If it exists at all I guess we do want to remove it -- if it's a dir, symlink, whatever, it has to be removed and we have to ask them. 

This is  harder than I realized.